streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorNode.java

```diff
@@ -86,16 +86,18 @@ public void process(final K key, final V value) {
      * @return a string representation of this node, useful for debugging.
      */
     public String toString() {
-        StringBuilder sb = new StringBuilder("");
-        sb.append(name + ": ");
+        StringBuilder sb = new StringBuilder(name);
+        sb.append(":");
         if (stateStores != null && !stateStores.isEmpty()) {
-            sb.append("stateStores [");
-            for (String store : (Set<String>) stateStores) {
-                sb.append(store + ",");
+            sb.append("\n\t\t\t\t\tstates:\t\t[");
```

Comment:
Looks like a lot of taps...  Should we have a "indent level" variable that we track and increase the indent level while calling nested `toString()` to simplify this?